### ISSUE TITLE
`localectl` can't find the `xkb` x11-keymap files

### ISSUE THREAD

**Issue by ahydronous** [NixOS/nixpkgs#345957]
### Describe the bug
`localectl` is unable to locate keymap files.

`localectl list-x11-keymap-`+`models`, `layouts`, `variants` all result in the error `Failed to open keyboard mapping list. No such file or directory`

### Steps To Reproduce
1.`localectl list-x11-keymap-layouts`

### Expected behavior
A list of the available keyboard layouts etc. should be printed.

### Additional context
This is probably a problem with an absolute path vs nixOS its folder structure

### Notify maintainers
@flokli
@kloenk 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.0, NixOS, 24.11 (Vicuna), 24.11pre681973.99dc8785f6a0`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - channels(root): `"nixos, unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by libewa** [comment]
yup. this also applies to `list-x11-keymap-variants`. however, completion for `setxkbmap` works.
`localectl set-x11-keymap` correctly errors about not being able to change settings on nixos
